Documentation/user-guides/alerting.md

```diff
@@ -205,72 +164,6 @@ will be a global AlertmanagerConfig. When the operator generates the
 Alertmanager configuration from it, the namespace label will not be enforced
 for routes and inhibition rules.
 
-## Exposing the Alertmanager service
-
-To access the Alertmanager interface, you have to expose the service to the outside. For
-simplicity, we use a `NodePort` Service.
-
-```yaml mdox-exec="cat example/user-guides/alerting/alertmanager-example-service.yaml"
-apiVersion: v1
-kind: Service
-metadata:
-  name: alertmanager-example
-spec:
-  type: NodePort
-  ports:
-  - name: web
-    nodePort: 30903
-    port: 9093
-    protocol: TCP
-    targetPort: web
-  selector:
-    alertmanager: example
-```
-
-Once the Service is created, the Alertmanager web server is available under the
-node's IP address on port `30903`.
-
-> Note: Exposing the Alertmanager web server this way may not be an applicable solution. Read more about the possible options in the [Ingress guide](exposing-prometheus-and-alertmanager.md).
-
-## Integrating with Prometheus
-
-### Configuring Alertmanager in Prometheus
-
-This Alertmanager cluster is now fully functional and highly available, but no
-alerts are fired against it.
-
-First, create a Prometheus instance that will send alerts to the Alertmanger cluster:
-
-```yaml mdox-exec="cat example/user-guides/alerting/prometheus-example.yaml"
-apiVersion: monitoring.coreos.com/v1
-kind: Prometheus
-metadata:
-  name: example
-spec:
-  serviceAccountName: prometheus
-  replicas: 2
-  alerting:
-    alertmanagers:
-    - namespace: default
-      name: alertmanager-example
-      port: web
-  serviceMonitorSelector:
-    matchLabels:
-      team: frontend
-  ruleSelector:
-    matchLabels:
-      role: alert-rules
-      prometheus: example
-```
-
-The `Prometheus` resource discovers all of the Alertmanager instances behind
-the `Service` created before (pay attention to `name`, `namespace` and `port`
-fields which should match with the definition of the Alertmanager Service).
-
-Open the Prometheus web interface, go to the "Status > Runtime & Build
-Information" page and check that the Prometheus has discovered 3 Alertmanager
-instances.
-
 ### Deploying Prometheus Rules
```

Comment:
After reviewing the page in the [preview](https://deploy-preview-104--sleepy-hopper-0fdb6b.netlify.app/docs/developer/alerting/#deploying-prometheus-rules), we noticed how Deploying Prometheus Rules doesn't fit with "Alerting Routes".

We probably want to move this to a separate page called "Alerting", covering more examples, e.g. Thanos Ruler. The YAML examples can also be simplified.